webview-ui/src/components/chat/ChatView.tsx

```diff
@@ -464,9 +464,10 @@ const ChatView = ({ isHidden, showAnnouncement, hideAnnouncement, showHistoryVie
 		startNewTask()
 	}, [startNewTask])
 
-	const { selectedModelInfo } = useMemo(() => {
+	const normalized = useMemo(() => {
 		return normalizeApiConfiguration(apiConfiguration)
 	}, [apiConfiguration])
+	const selectedModelInfo = normalized.selectedModelInfo
```

Comment:
Can we leave this as it was with destructuring it? This adds functionally nothing but makes pulling in changes from upstream potentially harder

Reply:
I rolled back this as well